equistore-core/CMakeLists.txt

```diff
@@ -209,27 +206,18 @@ set_target_properties(equistore::shared PROPERTIES
     IMPORTED_LOCATION ${EQUISTORE_SHARED_LOCATION}
     INTERFACE_INCLUDE_DIRECTORIES ${EQUISTORE_INCLUDE_DIR}
 )
+target_compile_features(equistore::shared INTERFACE cxx_std_11)
 
 set_target_properties(equistore::static PROPERTIES
     IMPORTED_LOCATION ${EQUISTORE_STATIC_LOCATION}
     INTERFACE_INCLUDE_DIRECTORIES ${EQUISTORE_INCLUDE_DIR}
 )
+target_compile_features(equistore::static INTERFACE cxx_std_11)
 
-if (${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.11)
-    if (BUILD_SHARED_LIBS)
-        add_library(equistore ALIAS equistore::shared)
-    else()
-        add_library(equistore ALIAS equistore::static)
-    endif()
+if (BUILD_SHARED_LIBS)
+    add_library(equistore ALIAS equistore::shared)
 else()
-    # CMake 3.10 (default on Ubuntu 20.04) does not support ALIAS for IMPORTED
```

Comment:
Was that comment wrong? Was this referring to Ubuntu 18.04?

Reply:
yes, this should have been ubuntu 18.04